Bang/Players/Player.cs

```diff
@@ -265,5 +265,11 @@ public void DropAllCards()
                 DropActiveCard(activeCard);
             }
         }
+
+        public Response EndTurn()
```

Comment:
I'm not sure that method `EndTurn()` should live in `Player`. It looks like `Gameplay` will be a better choice. 
Also I am confused by the name of the method and its body

Reply:
done